### PR TITLE
Support python client

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/client/py/nebula/client.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2018 vesoft inc. All rights reserved.
```
Comment (zhangguoqing) [inline src/client/py/nebula/client.py]: 2019 yet?

File: src/client/py/nebula/client.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+from thrift import Thrift
+from thrift.transport import TSocket
+from thrift.transport import TTransport
+from thrift.protocol import TBinaryProtocol
+
+from graph import GraphService
+
+class AuthException(Exception):
+    def __init__(message):
+        Exception.__init__ (self, message)
```
Comment (zhangguoqing) [inline src/client/py/nebula/client.py]: How about ```Exception.__init__(self, message)```?   Why leave a blank space?

File: src/client/py/nebula/client.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        self.protocol = TBinaryProtocol.TBinaryProtocol(self.transport)
+        self.transport.open()
+        self.client = GraphService.Client(self.protocol)
+
+    def authenticate(self, user, password):
+        authResponse = self.client.authenticate(user, password)
+        if authResponse.error_code == 0:
+            return authResponse.session_id
+        else:
+            raise AuthException("Auth failed")
```
Comment (zhangguoqing) [inline src/client/py/nebula/client.py]: How about below?
```
       if authResponse.error_code:
            raise AuthException("Auth failed")
        else:
            return authResponse.session_id
```

File: src/client/py/nebula/client.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+
+client = GraphClient("127.0.0.1", 13699)
+session = client.authenticate("user", "password")
+print "Session: " + str(session)
+
+executionResponse = client.execute(session, "show spaces")
+print executionResponse
+
+client.signout(session)
```
Comment (zhangguoqing) [inline src/client/py/nebula/client.py]: Why not move them to the tests.py?

File: src/interface/meta.thrift [hunk 5]
```diff
@@ -7,6 +7,7 @@
 namespace cpp nebula.meta
 namespace java com.vesoft.nebula.meta
 namespace go nebula.meta
+namespace py nebula.meta
```
Comment (zhangguoqing) [inline src/interface/meta.thrift]: I'm not sure this line is necessary?

Comment (darionyaphet) [reply src/interface/meta.thrift]: I will move it into example later :)

File: src/client/py/nebula/client.py (hunk 4 above)
Comment (darionyaphet) [reply src/client/py/nebula/client.py]: I will move it into example later :)

**Comment by yixinglu** [comment]
Move this client to the repo [vesoft-inc/nebula-python](https://github.com/vesoft-inc/nebula-python)

**Pull request closed without merging**